src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const formatedCSSData = {};
+  const splitedSource = sourceString.split(';');
+
+  splitedSource
+    .map((item) => {
+      const formatedPropAndValueObject = item
+        .split(':')
+        .map((elem) => elem.trim());
+
+      return formatedPropAndValueObject;
+    })
+    .forEach(([property, value]) => {
+      Object.assign(formatedCSSData, { [property]: value });
```

Comment:
The use of Object.assign() is unnecessary here and can be simplified. Since you are adding a single property to the 'formatedCSSData' object, you can directly assign the property using the bracket notation: 'formatedCSSData[property] = value'.